### ISSUE TITLE
Colour Palettes and Construction guidelines - Future colour font formats

### ISSUE THREAD

**Issue by mdtauk** [microsoft/fluentui-emoji#24]
I understand these may be free for remixing and use - and not for seeking people to submit additions.

But it would be neat to see the palettes, gradients, colour combinations used for these - as well as some of the guidelines the Emoji teams work to, when making future additions to the sets.

As for the font formats - will the upcoming Google colour font format, with its support for gradients - be used for generating from the non flat SVGs?  And is it possible the Windows team could adopt this, either as a replacement for the flat colour versions, or maybe as a toggle in settings?

**Comment by jasoncuster** [comment]
Hey hey @mdtauk - good to see you enjoying the emoji! Just like the icons, we'll add some guidelines soon - just updating some stuff. You know how it goes...WIP and all that...haha.

Thanks for the feedback on the gradient font support, that is a really interesting idea.

**Comment by PeterCon** [comment]
>... will the upcoming Google colour font format...

@mdtauk: What you're referring to isn't a Google specific format but is part of the [OpenType font formation specification](https://docs.microsoft.com/en-us/typography/opentype/spec/colr), developed jointly by Google, Microsoft and other contributors.

**Comment by mdtauk** [comment]
> >... will the upcoming Google colour font format...
> 
> @mdtauk: What you're referring to isn't a Google specific format but is part of the [OpenType font formation specification](https://docs.microsoft.com/en-us/typography/opentype/spec/colr), developed jointly by Google, Microsoft and other contributors.
> 
> Related: #16

Yes I know it's a joint effort by the opentype group.  But I think Google Fonts has been the most vocal in support and have implemented the Android 12 emojis with it.

I didn't mean to imply it was solely Google's endeavour

**Comment by mdtauk** [comment]
From the latest Windows insider builds, the emojis are moving to the new font format.
